exasol_script_languages_container_tool/lib/tasks/test/test_runner_db_test_task.py

```diff
@@ -19,6 +27,26 @@
 from exasol_integration_test_docker_environment.lib.test_environment.spawn_test_environment import SpawnTestEnvironment
 
 
+class DummyExecutor(DbOsExecutor):
+
+    def exec(self, cmd: str) -> ExecResult:
+        raise RuntimeError("Not supposed to be called.")
+
+    def prepare(self):
+        pass
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, type_, value, traceback):
+        pass
+
+
+class DummyExecFactory(DbOsExecFactory):
+    def executor(self) -> DbOsExecutor:
+        return DummyExecutor()
```

Comment:
@tkilias @ckunki pls let me know what you think about this.

Reply:
Please create a ticket in the ITDE, such that the EnvironmentInfo returns all necessary information. Add a comment, that the DummyExecutor is a workaround for this ticket.

Reply:
I created https://github.com/exasol/integration-test-docker-environment/issues/403 and https://github.com/exasol/script-languages-container-tool/issues/214

I realized that the missing info cannot be fetched by the ITDE itself, but rather needs to be forwared as a parameter.